internal/datacoord/index_meta.go

```diff
@@ -209,6 +209,7 @@ func checkParams(fieldIndex *model.Index, req *indexpb.CreateIndexRequest) bool
 		for i, param2 := range req.GetUserIndexParams() {
 			if param2.Key == param1.Key && param2.Value == param1.Value {
 				exist = true
+				break
```

Comment:
why?

Reply:
have to admit this logic is not well-written, but we should not break here, need to check all the params and make sure all the params are the same

Reply:
> have to admit original logic is not well-written, but we should not break here, need to check all the params and make sure all the params are the same

acutally, outside have circle 
![image](https://github.com/user-attachments/assets/775efb46-4ba3-4dc1-8f05-c8dbd5d5b3cb)
break is just break current key, outside keys always do this